docs/deployment/README.md#cumulus-instance-sizing

```diff
@@ -552,7 +552,7 @@ You should be able to visit the dashboard website at `http://<prefix>-dashboard.
 
 ## Cumulus Instance Sizing
 
-The Cumulus deployment default sizing for Elasticsearch instances, EC2 instances, and Autoscaling Groups are small and designed for testing and cost savings. The default settings are likely not suitable for production workloads. Sizing his highly individual and dependent on expected load and archive size.
+The Cumulus deployment default sizing for Elasticsearch instances, EC2 instances, and Autoscaling Groups are small and designed for testing and cost savings. The default settings are likely not suitable for production workloads. Sizing this is highly individual and dependent on expected load and archive size.
```

Comment:
Thanks for the correction! After review, we'd prefer the following phrasing. Could you make this change and request re-review?

```suggestion
The Cumulus deployment default sizing for Elasticsearch instances, EC2 instances, and Autoscaling Groups are small and designed for testing and cost savings. The default settings are likely not suitable for production workloads. Sizing is highly individual and dependent on expected load and archive size.
```